aptos-move/block-executor/src/executor.rs

```diff
@@ -1735,9 +1765,9 @@ where
                 init_speculative_logs(signature_verified_block.len());
```

Comment:
do we also need to flush it here? we might.

Reply:
Yes, below?

Reply:
sure